data/oecd/oecd_tariffs.ipynb

```diff
@@ -124,30 +124,29 @@
     "# Load country mapping to ISO code\n",
     "metadata_dir = \"metadata\"\n",
     "path_to_country_conversion_table = os.path.join(metadata_dir, \"country_name_conversion.csv\")\n",
+    "missing_countries = []\n",
     "if not os.path.exists(path_to_country_conversion_table):\n",
-    "  # get world bank country names and iso alpha3 code\n",
-    "  countries_table = pd.read_csv('../worldbank/processed/ccdr.csv')\n",
-    "  countries_table = countries_table[['country_code','country_name']].drop_duplicates().reset_index(drop=True)\n",
-    "  countries_table.columns = ['country_code',\t'wb_country_name']\n",
+    "    # get world bank country names and iso alpha3 code\n",
+    "    countries_table = pd.read_csv('../worldbank/processed/ccdr.csv')\n",
+    "    countries_table = countries_table[['country_code','country_name']].drop_duplicates().reset_index(drop=True)\n",
+    "    countries_table.columns = ['country_code',\t'wb_country_name']\n",
     "\n",
-    "  # get oecd country names\n",
-    "  # oecd_country_names = pd.read_csv('processed/fit.csv')\n",
-    "  oecd_country_names = list(df.country.unique())\n",
+    "    # get oecd country names\n",
+    "    # oecd_country_names = pd.read_csv('processed/fit.csv')\n",
+    "    oecd_country_names = list(df.country.unique())\n",
     "\n",
-    "  # match each oecd country name w/ wb country names\n",
-    "  # and store the oecd country name that do not match\n",
-    "  missing_countries = []\n",
```

Comment:
I've deleted this line - it was re-defining `missing_countries`.
Note the diff is easier to see with GitHub's `Hide whitespace` option turned on.